Frameworks/UIKit/UIScrollView.mm

```diff
@@ -602,16 +607,19 @@ - (void)setScrollEnabled:(BOOL)enable {
 }
 
 - (void)_setScrollEnabled:(BOOL)enable {
-    _scrollViewer.HorizontalScrollBarVisibility(enable ? _previousHorizontalScrollBarVisibility : Controls::ScrollBarVisibility::Disabled);
-    _scrollViewer.VerticalScrollBarVisibility(enable ? _previousVerticalScrollBarVisibility : Controls::ScrollBarVisibility::Disabled);
+    _scrollViewer.HorizontalScrollBarVisibility(enable ? _previousHorizontalScrollBarVisibility : Controls::ScrollBarVisibility::Hidden);
```

Comment:
So if I setShowsHorizontalScrollIndicator:YES, and then setScrollEnabled:NO, the scrollbars disappear.  Is that expected/intentional?